tests/test_storageclass_invalid.py

```diff
@@ -0,0 +1,65 @@
+import logging
+import os.path
+
+from resources.ocs import OCS
+from tests import helpers
+from ocs import constants, defaults
+from ocs.exceptions import TimeoutExpiredError
+
+
+logger = logging.getLogger(__name__)
+
+TEMPLATE_DIR = os.path.join('templates', 'ocs-deployment')
+PVC_TEMPLATE = os.path.join(TEMPLATE_DIR, 'PersistentVolumeClaim.yaml')
```

Comment:
This seems to be not used anywhere, so maybe the file `PersistentVolumeClaim.yaml` is also spare?